addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -13,6 +13,8 @@ class PaymentAcquirerStripe(models.Model):
 
     stripe_secret_key = fields.Char(required_if_provider='stripe')
     stripe_publishable_key = fields.Char(required_if_provider='stripe')
+    image_url = fields.Char("Image URL", required_if_provider='stripe',
```

Comment:
Well adding a char field expecting an image gives you no guarantee either of size and I just tested, it correctly resize it.
By the way, thanks to my recent commit 43f619ed761be3acd515fe953ca7848db0c86074, your module will force you to have an image. So I think you need to remove the `required_if_provider`.